packages/core/admin/admin/src/pages/SettingsPage/pages/ApplicationInfosPage/utils/parseFileMetadatas.js

```diff
@@ -25,12 +25,15 @@ const getFileDimensions = file => {
   });
 };
 
-const rawFileToAsset = rawFile => {
+const rawFileToAsset = (rawFile, fileDimensions) => {
   return {
+    ext: rawFile.name.split('.').pop(),
```

Comment:
Is there any test for this file/function?

Reply:
Yes you can find the entire parseFileMetadatas flow tested inside `LogoInput/tests/index.test.js`!